clients/da-vinci-client/src/main/java/com/linkedin/davinci/schema/writecompute/WriteComputeHandlerV2.java

```diff
@@ -166,10 +169,20 @@ private GenericRecord assertAndGetTsRecordFieldIsRecord(GenericRecord timestampR
     if (!(fieldTimestamp instanceof GenericRecord)) {
       throw new IllegalStateException(
           String.format(
-              "Expect field %s in the timestamp record to be a generic record. Got timestamp record: %s",
-              fieldTimestamp,
+              "Expect field '%s' in the timestamp record to be a generic record. Got timestamp record: %s",
+              fieldName,
               timestampRecord));
     }
     return (GenericRecord) fieldTimestamp;
   }
+
+  // protected access for testing
+  public Object updateArray(Schema arraySchema, List originalArray, Object writeComputeArray) {
```

Comment:
Should this be `protected`?

Reply:
Yeah, this is used in client-common's unit test but I think that test class it moved to DVC package now, so it can be "protected" or even default (no modifier)

Reply:
Thanks for catching this... I did this in an earlier iteration of the refactoring, but since then it's not needed anymore. I reverted this change.